docs/metrics.md

```diff
@@ -201,6 +201,21 @@ Metrics for the [Four Golden Signals](https://sre.google/sre-book/monitoring-dis
     Some metric attributes may have high cardinality and are marked with ⚠️ to warn you. You may need to disable this metric or disable the attribute.
 <!-- titles should be the exact metric name for deep-linking, alphabetical ordered -->
 <!-- titles are without argo_workflows prefix -->
+#### `controller_build_info`
+
+The build information for this workflow controller
+
+| attribute   | explanation                                                            |
+|-------------|------------------------------------------------------------------------|
+| `version`   | The version of argo-workflows                                          |
+| `platform`  | Platform this is running on, as go describes it e.g. `linux/amd64` |
+| `gover`     | Version of go that built this workflow controller                  |
+| `build`     | Build date for this workflow controller                                |
+| `compiler`  | The compiler used to build this workflow controller e.g. `gc`          |
+| `commit`    | The full git SHA1 of the code that built this workflow controller      |
+| `treestate` | Whether the git tree was `dirty` or `clean` when this was built        |
+| `tag`       | The tag on the git commit or `untagged` if it was not tagged           |
```

Comment:
```suggestion
| `git_commit`    | The full Git SHA1 commit      |
| `git_tree_state` | Whether the Git tree was `dirty` or `clean` when built        |
| `git_tag`       | The Git tag or `untagged` if it was not tagged           |
```

- match `argo version` and split words with underscores per OTel convention
- simplify
- "Git" as proper noun

Reply:
Done